essence-of-live-coding-frp/src/LiveCoding/FRP/Curry.hs

```diff
@@ -0,0 +1,43 @@
+{-# LANGUAGE FlexibleContexts #-}
+
+-- |
+-- Module      : LiveCoding.FRP.Curry
+-- Description : Function uncurrying utilities
+-- Copyright   : (c) 2022 Miguel Negrão
+-- Maintainer  : miguel.negrao@friendlyvirus.org
+-- Stability   : experimental
+-- Portability : POSIX, Windows
+--
+-- Functions which are lifted into arrows have to be in completely curried form.
+-- This module provides utility functions to automatically uncurry functions and
+-- lift them to 'Arrow's using the tuple package's 'Curry' typeclass.
+module LiveCoding.FRP.Curry where
```

Comment:
This should be part of either https://github.com/miguel-negrao/arrow-utils or directly of `essence-of-live-coding`.

Reply:
It has some functions which depend on Cell. Should I put the functions that depend only on Arrow in arrow-utils, and the ones that depend on Cell directly in essence-of-live-coding ?

Reply:
Ah I see. Yes, that sounds like the right way to go forward.